### PR TITLE
Add rumbleBrieflyCommand when intaking

### PR DESCRIPTION
**Pull request by tails618**
Rumbles the controller when the driver can release the button after intaking

### CODE REVIEW

File: src/main/java/frc/robot/commands/IntakeSequence.java [hunk 1]
```diff
@@ -41,12 +48,11 @@ public IntakeSequence(Intake intake, Elevator elevator, Conveyor conveyor, Shoot
...
-            () -> {
-              intake.stopRollers();
-            },
-            intake),
-        new InstantCommand(() -> intake.setDesiredIntakePosition(IntakePosition.STOWED)));
+        new ParallelCommandGroup(
+            Conveyor.receive(conveyor),
+            new WaitUntilCommand(() -> intake.beamBreakSensor.get())), // TODO test true or false
+        new InstantCommand(intake::stopRollers, intake),
+        new InstantCommand(() -> intake.setDesiredIntakePosition(IntakePosition.STOWED)),
```
Comment (balghane) [inline src/main/java/frc/robot/commands/IntakeSequence.java]: I don't think we want to require the beam break to be triggered. If it gets into the conveyor then we should also be done. How about 
```suggestion
        new ParallelDeadlineGroup(
            Conveyor.receive(conveyor),
            new SequentialCommandGroup(
                new WaitUntilCommand(() -> intake.beamBreakSensor.get())), // TODO test true or false
                new InstantCommand(intake::stopRollers, intake),
                new InstantCommand(() -> intake.setDesiredIntakePosition(IntakePosition.STOWED)),
            )),
        new InstantCommand(intake::stopRollers, intake),
        new InstantCommand(() -> intake.setDesiredIntakePosition(IntakePosition.STOWED)),
```

File: src/main/java/frc/robot/commands/autos/OneFiveLeave.java [hunk 2]
```diff
@@ -23,17 +24,19 @@ public OneFiveLeave(
...
     final boolean FIELD_RELATIVE = true;
     final double WAIT_BEFORE_DRIVE_SEC = 1.0;
+    Command rumbleCommand = rumbleBrieflyCommand;
 
     addRequirements(intake, elevator, shooter, conveyor, drive);
     addCommands(
         new ScoreThisNote(intake, elevator, shooter, conveyor),
         new ParallelCommandGroup(
-            new IntakeSequence(intake, elevator, conveyor, shooter)
+            new IntakeSequence(intake, elevator, conveyor, shooter, rumbleCommand)
```
Comment (balghane) [inline src/main/java/frc/robot/commands/autos/OneFiveLeave.java]: Maybe make the auto ones a no-op? We don't want to rumble in auto

Comment (avakimc) [reply src/main/java/frc/robot/commands/autos/OneFiveLeave.java]: just wondering, what does "no-op" mean?

**Review by avakimc — COMMENTED** [review]
same comments as basel

File: src/main/java/frc/robot/commands/autos/OneFiveLeave.java (hunk 2 above)
Comment (balghane) [reply src/main/java/frc/robot/commands/autos/OneFiveLeave.java]: no operation. Pass in an empty command so it doesn't rumble in auto

File: src/main/java/frc/robot/commands/IntakeSequence.java (hunk 1 above)
Comment (voidSevenSevenSix) [reply src/main/java/frc/robot/commands/IntakeSequence.java]: What Basel has written looks correct

Comment (tails618) [reply src/main/java/frc/robot/commands/IntakeSequence.java]: If I'm correct about where it is, the sensor will be triggered before `Conveyor.receive(conveyor)` is done, so I believe the intake will stop and go to stowed position prematurely?

File: src/main/java/frc/robot/commands/autos/OneFiveLeave.java (hunk 2 above)
Comment (tails618) [reply src/main/java/frc/robot/commands/autos/OneFiveLeave.java]: Changed

File: src/main/java/frc/robot/commands/IntakeSequence.java (hunk 1 above)
Comment (balghane) [reply src/main/java/frc/robot/commands/IntakeSequence.java]: Intake can go in before conveyor is fully done

**Pull request closed without merging**